src/control/nav2_controller_example/src/dwa_controller.cpp

```diff
@@ -61,4 +64,16 @@ DwaController::execute(const nav2_tasks::FollowPathCommand::SharedPtr /*command*
   return TaskStatus::SUCCEEDED;
 }
 
+void DwaController::sendVelocity(double speed, double angle)
```

Comment:
Speed and angle should not be 1 dimensional.

Reply:
You are correct. I've removed the angle parameter for now since the only thing actually working is X velocity. I'll expand this function's interface as we add more capability to this controller.

Reply:
Sounds good.